composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "norzechowicz/aceeditor-bundle",
+    "name": "sukovanej/aceeditor-bundle",
```

Comment:
you are joking, right?

Reply:
Sory, my fault... Didnt know these changes will get to the pull request.

Reply:
sure, no problem. If you want this feature to get merged please open PR against master branch just with required changes and if possible test case